docs/blog/2023-02-17-how-to-use-drand.md

```diff
@@ -119,16 +119,18 @@ const chain = new HttpCachingChain("https://api.drand.sh")
 const client = new HttpChainClient(chain)
 fetchBeaconByTime(client, Date.now()).then(beacon => {   
 
-	const flip = parseInt(Number("0x" + beacon.randomness)) % 2    
+	const flip = BigInt("0x" + beacon.randomness) % BigInt(2)
 
-	if (flip === 0) {     
+	if (flip === 0) {      
```

Comment:
I swear husky keeps adding this damn random whitespace

Reply:
Check out https://editorconfig.org/ to remove them all on save